### PR TITLE
Introduction fixes

### PR DESCRIPTION
**Pull request by m0mus**

### CODE REVIEW

File: docs/about/introduction.adoc [hunk 1]
```diff
@@ -16,102 +16,120 @@
...
+* <<Prerequisites, Prerequisites>>
+* <<Next Steps, Next Steps>>
 
-Helidon provides an open source, lightweight, fast, reactive, cloud native framework for developing Java microservices. Helidon implements and supports MicroProfile, a baseline platform definition that leverages Java EE and Jakarta EE technologies for microservices and delivers application portability across multiple runtimes.
+== What is Helidon?
 
+link:https://helidon.io[Helidon] is a collection of Java libraries for writing microservices that run on a fast web core powered by link:https://netty.io[Netty]. It's small, fast and fun to use. 
 
-== Using Cloud-native Tools with Helidon
+Helison is open source under Alache 2.0 license. Sources are available on link:{helidon-github-tree-url}[GitHub].
```
Comment (tomas-langer) [inline docs/about/introduction.adoc:38]: Helison -> Helidon

File: docs/about/introduction.adoc [hunk 2]
```diff
@@ -16,102 +16,120 @@
...
 
+== Helidon Flavors
 
-== Understanding the Helidon Frameworks
-Helidon supports two programming models for writing microservices: *Helidon SE* and *Helidon MP*.
-
-SE is designed to be a microframework that supports the reactive programming model, while Helidon MP is an Eclipse MicroProfile runtime that allows the Jakarta EE community to run microservices in a portable way.
-
-The table below shows to primary differences between Helidon SE and Helidon MP.
+Helidon comes in two flavors: *Helison SE* and *Helidon MP*. Think about flavors as frameworks providing similar functionality using differet development experience.
```
Comment (tomas-langer) [inline docs/about/introduction.adoc:48]: Helison -> Helidon

File: docs/about/introduction.adoc [hunk 3]
```diff
@@ -16,102 +16,120 @@
...
-Helidon provides an open source, lightweight, fast, reactive, cloud native framework for developing Java microservices. Helidon implements and supports MicroProfile, a baseline platform definition that leverages Java EE and Jakarta EE technologies for microservices and delivers application portability across multiple runtimes.
+== What is Helidon?
 
+link:https://helidon.io[Helidon] is a collection of Java libraries for writing microservices that run on a fast web core powered by link:https://netty.io[Netty]. It's small, fast and fun to use. 
 
-== Using Cloud-native Tools with Helidon
+Helison is open source under Alache 2.0 license. Sources are available on link:{helidon-github-tree-url}[GitHub].
 
-Helidon libraries interoperate with popular tools from the cloud-native space, so there's no need for any specific tooling or deployment model. Helidon can be used with:
+Helidon is cloud-native ready. It provides fast start-up time, has low memory consumption and small disk footprint. It also comes with full observability stack out of the box including Health Checks, Metrics , Tracing and Logging.
```
Comment (tomas-langer) [inline docs/about/introduction.adoc:40]: "Metrics ," -> "Metrics, "

File: docs/mp/introduction.adoc [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+| xref:{rootdir}/mp/beanvalidation.adoc[Jakarta Bean Validation]
+| link:{jakarta-bean-validation-spec-url}[{version-lib-jakarta-bean-validation}]
+| Object level constraint declaration and validation facility
+
+| Jakarta Context and Dependency Injection (CDI)
+| link:{jakarta-cdi-spec-url}[{version-lib-jakarta-cdi}]
+| Declarative dependency injection and supporting services
+
+| Jakarta JSON Processing (JSONP)
```
Comment (tomas-langer) [inline docs/mp/introduction.adoc:43]: I think we use JSON-P and JSON-B

File: docs/mp/introduction.adoc [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+
+| xref:{rootdir}/mp/jwt.adoc[MicroProfile JWT Auth]
+| link:{microprofile-jwt-spec-url}[{version-lib-microprofile-jwt}]
+| Defines a compact and self-contained way for securely transmitting information between parties as a JSON object
+
+| xref:{rootdir}/mp/lra.adoc[MicroProfile Long-Running Actions (LRA)]
+| link:{microprofile-lra-spec-url}[{version-lib-microprofile-lra-api}]
+| Distributed transactions for microservices following SAGA pattern
+
+| xref:{rootdir}/mp/metrics/metrics.adoc[MicroProfile Metrics]
```
Comment (tomas-langer) [inline docs/mp/introduction.adoc:99]: Metrics is a library to add metrics to your application and expose them either in Prometheus or MP JSON format

File: docs/mp/introduction.adoc [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+| link:{microprofile-metrics-spec-url}[{version-lib-microprofile-metrics-api}]
+| Telemetry data in Prometheus format
+
+| xref:{rootdir}/mp/openapi.adoc[MicroProfile Open API]
+| link:{microprofile-reactive-messaging-spec-url}[{version-lib-microprofile-reactive-messaging-api}]
+| Tracking service calls in distributed environment
```
Comment (tomas-langer) [inline docs/mp/introduction.adoc:105]: Open API is documentation of microservice endpoints

File: docs/mp/introduction.adoc [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+== Other Components
+
+!===
+| Component | Description
+
+| xref:{rootdir}/mp/cors/cors.adoc[CORS]
+| Cross Origin Resource Sharing – API to control if and how REST resources served by their applications can be shared across origins 
+
+| xref:{rootdir}/mp/grpc/server-side-services.adoc[gRPC]
+| gRPC servers and clients.
```
Comment (tomas-langer) [inline docs/mp/introduction.adoc:134]: gRPC server and client (Not plural)

File: docs/sitegen.yaml [hunk 8]
```diff
@@ -52,26 +52,20 @@ backend:
...
-                  value: "assistant"
-                sources:
-                  - "overview.adoc"
-                  - "introduction.adoc"
-                  - "prerequisites.adoc"
-                  - "managing-dependencies.adoc"
-                  - "windows.adoc"
-                  - "cli.adoc"
-                  - "kubernetes.adoc"
-                  - "upgrade.adoc"
```
Comment (tomas-langer) [inline docs/sitegen.yaml:74]: Where is our upgrade doc going to be located?
Also I am missing SE introduction page (this change only contains MP)

**Review by tomas-langer — CHANGES_REQUESTED** [review]

File: docs/about/introduction.adoc [hunk 9]
```diff
@@ -16,102 +16,120 @@
...
 :rootdir: {docdir}/..
 
 include::{rootdir}/includes/attributes.adoc[]
 
-Helidon is a collection of Java libraries for writing microservices that run on a fast web core powered by Netty. Its available in two frameworks: Helidon SE and Helidon MP.
+== Contents
 
-== A Collection of Java Libraries
+* <<What is Helidon?, What is Helidon?>>
+* <<Helidon Flavors, Helidon Flavors>>
```
Comment (barchetta) [inline docs/about/introduction.adoc:29]: In 2.x documentation we never used "flavor". We used "programming model" which I prefer.

File: docs/sitegen.yaml [hunk 10]
```diff
@@ -52,26 +52,20 @@ backend:
         type: "image"
         value: "images/helidon_logo_dark.svg"
       items:
+        - type: "PAGE"
+          title: "Introduction"
+          source: "about/introduction.adoc"
+          glyph:
+            type: "icon"
+            value: "widgets"
+        - type: "PAGE"
```
Comment (barchetta) [inline docs/sitegen.yaml:61]: We should have a top level"Get Started" page here with some version of what is currently in (the misnamed) `prerequisites.adoc`. It's important that we have "Get Started" very visible at the top level like we have in 2.x.

File: docs/about/introduction.adoc [hunk 11]
```diff
@@ -16,102 +16,120 @@
...
+- https://maven.apache.org/download.cgi[Maven 3.8.6+]
+- https://docs.docker.com/install/[Docker 18.09+]
+- https://kubernetes.io/docs/tasks/tools/install-kubectl/[Kubectl 1.16.5+]
+
+We are also strongly suggesting installing xref:cli.adoc[Helidon CLI] (Command Line Interface) which helps generating and building Helidon projects.
 
 == Next Steps
 
-Try the xref:prerequisites.adoc[Helidon Quickstart Examples].
+Next would be to choose a Helidon flavor and start using it. Check the following
```
Comment (barchetta) [inline docs/about/introduction.adoc:132]: This should point to a top level "Get Started" page like we do in 2.x (. See my comment on `sitegen.yaml`). For example, replace this section with just:

Try the xref:{rootdir}/about/getstarted.adoc[Helidon Quickstart Examples]

File: docs/sitegen.yaml [hunk 12]
```diff
@@ -52,26 +52,20 @@ backend:
...
-            items:
-              - type: "MENU"
-                title: "About"
-                glyph:
-                  type: "icon"
-                  value: "assistant"
-                sources:
-                  - "overview.adoc"
-                  - "introduction.adoc"
-                  - "prerequisites.adoc"
```
Comment (barchetta) [inline docs/sitegen.yaml:69]: Some version of `prerequisistes.adoc` should be preserved in a top level `getstarted.adoc`

File: docs/about/introduction.adoc [hunk 13]
```diff
@@ -16,102 +16,120 @@
...
-* https://www.docker.com/[Docker]
-* https://kubernetes.io/[Kubernetes]
-* https://prometheus.io/[Prometheus]
-* https://opentracing.io/[OpenTracing]
-* https://coreos.com/etcd/[Etcd]
+Helidon fully supports GraalVM native-image allowing you to build native executable out of your Java application.
 
-TIP: The xref:prerequisites.adoc[Helidon Quickstart Examples]
- contain support for Docker and Kubernetes.
+Helidon has minimum number of third-party dependencies by design.
```
Comment (barchetta) [inline docs/about/introduction.adoc:44]: I would remove this line.

File: docs/mp/introduction.adoc (hunk 4 above)
Comment (m0mus) [reply docs/mp/introduction.adoc:43]: JSONP and JSONB are two different specifications. JSONB is listed in a row below.

File: docs/mp/introduction.adoc (hunk 5 above)
Comment (m0mus) [reply docs/mp/introduction.adoc:99]: Defining and exposing telemetry data in Prometheus and JSON formats

File: docs/about/introduction.adoc (hunk 1 above)
Comment (m0mus) [reply docs/about/introduction.adoc]: Fixed

File: docs/about/introduction.adoc (hunk 2 above)
Comment (m0mus) [reply docs/about/introduction.adoc]: Fixed

File: docs/about/introduction.adoc (hunk 3 above)
Comment (m0mus) [reply docs/about/introduction.adoc]: Fixed

File: docs/mp/introduction.adoc (hunk 6 above)
Comment (m0mus) [reply docs/mp/introduction.adoc]: Fixed

File: docs/sitegen.yaml (hunk 8 above)
Comment (m0mus) [reply docs/sitegen.yaml:74]: Mitia is working on migration guides. I expect adding links to the corresponding introduction pages as part of his commit.

Comment (m0mus) [reply docs/sitegen.yaml:74]: This PR doesn't cover SE introduction page. IMO, it doesn't need modifications.

File: docs/about/introduction.adoc (hunk 11 above)
Comment (m0mus) [reply docs/about/introduction.adoc:132]: We are going in direction of separating Helidon documentation into self-contained documentation for each flavor. My idea is that all QuickStart pages must be flavor specific. Having one page which explains how to add JAVA_HOME and contains links to SE and MP quickstart guides doesn't make sense to me. Users will go to SE or MP introductio and find proper QuickStart examples there.

File: docs/sitegen.yaml (hunk 10 above)
Comment (m0mus) [reply docs/sitegen.yaml:61]: We are going in direction of separating Helidon documentation into self-contained documentation for each flavor. My idea is that all QuickStart pages must be flavor specific. Having one page in the top level which explains how to add `JAVA_HOME` and contains links to SE and MP quickstart guides doesn't make sense to me.

File: docs/about/introduction.adoc (hunk 9 above)
Comment (m0mus) [reply docs/about/introduction.adoc:29]: We use _flavor_ in all presentations. Upcoming Nima will have the same programming model as MP (blocking with annotations). Using _programming model_  as differenciator will not make sense.

File: docs/sitegen.yaml (hunk 12 above)
Comment (m0mus) [reply docs/sitegen.yaml:69]: We are trying to compact the left panel. Prerequisities are not a part of introduction page and can be referenced as `http://xxx/about/introduction#_prerequisites)`

File: docs/about/introduction.adoc (hunk 13 above)
Comment (m0mus) [reply docs/about/introduction.adoc]: Removed

File: docs/mp/introduction.adoc (hunk 4 above)
Comment (m0mus) [reply docs/mp/introduction.adoc:43]: Ahhh... Fine, I'll change it this way.

Comment (m0mus) [reply docs/mp/introduction.adoc]: Done

File: docs/about/introduction.adoc (hunk 11 above)
Comment (barchetta) [reply docs/about/introduction.adoc:130]: The point is: we should have *one* page we can link to for getting started.  We need the "Get Started" button on the web site to go to one page. That one page might simply link to the other getting started pages -- but we need  one page. And if it is a top level, highly visible page all the better. This was a lesson we learned in 2.x. I would rather not repeat the same mistake again.

**Review by tomas-langer — APPROVED** [review]
OK with me, please resolve comments with @barchetta

File: docs/about/introduction.adoc (hunk 11 above)
Comment (m0mus) [reply docs/about/introduction.adoc:130]: Get Started page is back!

**Review by tomas-langer — APPROVED** [review]
LGTM

**Comment by m0mus** [comment]
Fixes #4526

**Pull request merged**